finance/community-updates/2020-10.txt

```diff
@@ -0,0 +1,68 @@
+Dear Astropy Community,
+
+Once again, the Interim Finance Committee would like to give you some updates
+on our activities over the couple of months.
+
+- The process for selecting Inclusion, Diversity, and Equity in Astropy
+  proposals has been completed [1]. We were delighted with the high-quality of the
+  proposals received, and with the community discussion which resulted. Thanks
+  to everybody who took part!
+
+  Based on this process, two proposals have been funded: one to fund an
+  Astropy intern through Outreachy [2] and the other to develop a peer
+  mentoring network for women of color and other marginalized groups [3].
+  We're really excited to see these projects make progress!
+
+  We very much hope to be able to issue further calls for proposals in this
+  area in the future. Further, we plan to use a development of the
+  collaborative selection process developed here for other funding decisions.
+  Given both of those, we'd love to hear feedback from the community on how
+  our process could be improved.
+
+  [1] https://groups.google.com/g/astropy-dev/c/NDU3dTDDsAc
+  [2] https://github.com/astropy/astropy-project/blob/master/finance/IDEA_2019_outreachy.md
+  [3] https://github.com/astropy/astropy-project/blob/master/finance/IDEA_2019_WoCCode.md
```

Comment:
Should it be obvious why these are named “IDEA_2019”?

Reply:
Not obvious, but see change suggestion above.

Reply:
My question was really about the “2019” rather than the “IDEA”. Given the call was issued in 2020, using funds earmarked for (US) FY20, I'm not sure where the 2019 is coming from. I'd like to rename these files for clarity, but I'm not sure if I missed something. @hamogu — you named them originally — could you clarify please?

Reply:
I think I'm just living a little bit (well a year) in the past. Should be 2020.